include/openthread/platform/radio.h

```diff
@@ -648,6 +648,8 @@ void otPlatRadioSetPromiscuous(otInstance *aInstance, bool aEnable);
  * If a platform supports `OT_RADIO_CAPS_RX_ON_WHEN_IDLE` it must also support `OT_RADIO_CAPS_CSMA_BACKOFF` and handle
  * idle periods after CCA as described above.
  *
+ * When @p `aEnable` is set to FALSE, it is expected that the radio platform will put the radio to sleep.
```

Comment:
What about:
```suggestion
 * When @p `aEnable` is set to FALSE, the radio will be put to sleep even if it's in receive state.
```

Reply:
Not sure about `even if` in suggested change. I think original text is more clear.


I would modify it as follows: 

Upon the transition of the "RxOnIdle" flag from `TRUE` to `FALSE`, the radio platform should enter sleep mode. If the radio is currently in receive mode, it should enter sleep mode immediately. Otherwise, it should enter sleep mode after the current operation is completed.

Reply:
SGTM.